tasks/rh-sign-image-cosign/README.md

```diff
@@ -8,8 +8,19 @@ Tekton task to sign container images in snapshot by cosign.
 |----------------|---------------------------------------------------------------------------|----------|---------------|
 | snapshotPath   | Path to the JSON string of the mapped Snapshot spec in the data workspace | No       | -             |
 | secretName     | Name of secret containing needed credentials                              | No       | -             |
+| signRegistryAccessPath | The relative path in the workspace to a text file that contains a list of repositories that needs registry.access.redhat.com image references to be signed (i.e. requires_terms=true), one repository string per line, e.g. "rhtas/cosign-rhel9". | No       | -                    |
```

Comment:
Can you fix the vertical alignment of the other parameters to match

Reply:
I knew this was coming :D But what's the point here? That description is just very long, so it doesn't seem to make sense to make the others also this wide - nobody has a monitor this wide anyway. And I don't think there's a way to have a multiline string in a markdown table.

Reply:
The point is the same point as consistent changelogs (like * vs -). Really makes no difference, just nice on the eyes :sunglasses:

Reply:
So do you really want to have all the lines this wide in this case? I can change it.

Reply:
I don't really care tbh :joy: but I think we aim to be consistent in our READMEs and so this should stay consistent as well and make the wide lines

Reply:
I think the goal of being consistent is improved readability. Here it actually makes it less readable IMO. Thank god there are plugins for this, otherwise I would kill myself :) Fixed.